nucliadb/nucliadb/search/search/merge.py

```diff
@@ -289,8 +295,13 @@ async def merge_vectors_results(
             index_int = int(index)
         except ValueError:
             index_int = -1
-        text = await get_text_sentence(
```

Comment:
Is the new `get_paragraph_text` also compatible with what `get_text_sentence` was doing? I see there is some sketchy logic here: https://github.com/nuclia/nucliadb/blob/6d3a47ac28baeeae160d4e62655da4c0e8279abe/nucliadb/nucliadb/search/search/fetch.py#L150-L152

Reply:
okay, for now, this is reverted just in case we don't understand this correctly.